bin/varnishd/cache/cache.h

```diff
@@ -117,6 +117,7 @@ struct cli;
 struct http_conn;
 struct listen_sock;
 struct mempool;
+struct ocstash;
```

Comment:
Using this line as an anchor to mention that you had a text editor accident with your first patch. This line leaked from the git commit template at the end of the first commit message:

> t Please enter the commit message for your changes. Lines starting